app/controllers/bookings_controller.rb

```diff
@@ -50,6 +53,10 @@ def set_booking
     @booking = Booking.find(params[:id])
   end
 
+  def set_bus
```

Comment:
Method name is incorrect, why we are taking it as set_bus when we are finding tain?